app/build.gradle

```diff
@@ -119,31 +111,22 @@ android {
     }
 }
 
-androidExtensions {
-    // Needed for Kotlin Parcelable support
-    experimental = true
-
-    // Prevents a bunch of other stuff from being generated
-    defaultCacheImplementation = "none"
-}
-
 ext {
     // App dependency versions
-    appCompatVersion = '1.0.2'
+    appCompatVersion = '1.4.0'
     supportVersion = '1.0.0'
-    playServicesVersion = '17.1.0'
+    playServicesVersion = '17.6.0'
     lifecycleVersion = '2.0.0'
-    koinVersion = '3.1.2'
-    retrofitVersion = '2.4.0'
-    okHttpVersion = '3.11.0'
-    leakcanaryVersion = '1.5.4'
-    moshiVersion = '1.12.0'
+    koinVersion = '3.1.4'
+    retrofitVersion = '2.9.0'
+    okHttpVersion = '5.0.0-alpha.3'
+    moshiVersion = '1.13.0'
 
     // Test dependency versions
-    mockitoVersion = '2.25.0'
-    robolectricVersion = '4.2'
-    androidTestSupportVersion = '1.1.0'
-    espressoVersion = '3.1.0-alpha4'
+    mockitoVersion = '4.1.0'
+    robolectricVersion = '4.7.3'
+    androidTestSupportVersion = '1.4.0'
+    espressoVersion = '3.5.0-alpha03'
```

Comment:
for consistency change all hard coded numbers into variables to match

Reply:
Replaced all hardcoded versions with variables